src/components/WebComponentProject/WebComponentProject.jsx

```diff
@@ -38,8 +44,7 @@ const WebComponentProject = ({
 
   dispatch(setIsSplitView(false));
   dispatch(setWebComponent(true));
-
-  console.log(`outputOnly: ${outputOnly}`);
+  dispatch(setIsOutputOnly(outputOnly));
```

Comment:
Apologies if my brain is scrambled 🍳  - it looks to me that calling `setIsOutputOnly` will always cause the [code here (line 197) ](https://github.com/RaspberryPiFoundation/editor-ui/pull/915/files#diff-3db14236c0b7d79bb1a84a935f0d541508cd0693a1c6e068faf3146420089504R197) to always set it to true even if the argument is false?